docs/xms-long-running-operation-property.md

```diff
@@ -0,0 +1,67 @@
+# XmsLongRunningOperationProperty
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Async-V1-15
+
+## Output Message
+
+If an operation's (PUT/POST/PATCH/DELETE) responses have `Location` or `Azure-AsyncOperation` headers then it MUST have the property `x-ms-long-running-operation` set to `true`.
+
+## Description
+
+If an operation's (PUT/POST/PATCH/DELETE) responses have `Location` or `Azure-AsyncOperation` headers then it MUST have the property `x-ms-long-running-operation` set to `true`.
+
+## How to fix the violation
+
+You can do either of the following:
+- Set the property `x-ms-long-running-operation` to `true`.
```

Comment:
> Set the property `x-ms-long-running-operation` to `true`. [](http://example.com/codeflow?start=1&length=58)

If the operation is intended to be a long running operation, set the property `x-ms-long-running-operation` to `true`.

Reply:
Added